nav2_bt_navigator/README.md

```diff
@@ -18,85 +18,70 @@ BtNavigator:
 
 Using the XML filename as a parameter makes it easy to change or extend the logic used for navigation. Once can simply update the XML description for the BT and the BtNavigator task server will use the new description.
 
+## Behavior Tree nodes
+
+A Behavior Tree consists of control flow nodes, such as fallback, sequence, parallel, and decorator, as well as two execution nodes: condition and action nodes. Execution nodes are the leaf nodes of the tree. When a leaf node is ticked, the node does some work and it returns either SUCCESS, FAILURE or RUNNING.  The current Navigation2 software implements a few custom nodes, including Conditions and Actions. The user can also define and register additional node types that can then be used in BTs and the corresponding XML descriptions.
+
 ## Navigation Behavior Trees
 
-The BT Navigator package has a few sample XML-based descriptions of BTs.
+The BT Navigator package has three sample XML-based descriptions of BTs.  These trees are [navigate_w_replanning.xml](behavior_trees/navigate_w_replanning.xml), [navigate_w_replanning_and_recovery.xml](behavior_trees/navigate_w_replanning_and_recovery.xml), and [auto_localization_w_replanning_and_recovery.xml](behavior_trees/auto_localization_w_replanning_and_recovery.xml).  The user may use any of these sample trees or develop a more complex tree which could better suit the user needs.
 
-### Simple sequential invocation of planning and control
+### Navigate with Replanning
 
-[Simple_sequential.xml](behavior_trees/simple_sequential.xml) implements a basic navigation by first computing the path and then following the path.
+[navigate_w_replanning.xml](behavior_trees/navigate_w_replanning.xml) implements basic navigation by continuously computing and updating the path at a rate of 1Hz and following the path at a rate of 10Hz.
```

Comment:
This sentence makes it sound like the behavior tree implements or specific the path following rate of 10Hz. How about something like "navigate_w_replanning.xml implements basic navigation by continuously computing and updating the path at a rate of 1Hz. The default local planner, the nav2_dwb_controller, implements path following at a rate of 10Hz." Either that, or drop the second part entirely.